app/src/components/pages/divisionsPage/divisionsPage.tsx

```diff
@@ -19,6 +19,11 @@ interface DivisionsProps {
 
 export default function DivisionsPage({ summary, divisions }: DivisionsProps) {
   const [isLoaded, setIsLoaded] = useState(false);
+  if ((window as any).newrelic) {
+    (window as any).newrelic.log("test log from divisions page", {
```

Comment:
Do you know where I can see this in NR? I see logs here https://one.newrelic.com/logger?account=121334&duration=86400000&filters=%28domain%20IN%20%28%27BROWSER%27%2C%20%27BROWSER%27%29%20AND%20type%20IN%20%28%27APPLICATION%27%2C%20%27MICRO_FRONTEND%27%29%29&state=97ecb909-d652-d0ee-7371-53a1dcc37c0e

but it's only the "test error from fetch API".  I was confused, I saw the following in NR but not in the repo and then noticed from the commits that you added and then removed it. From what I understand, you logged "test error from fetch API" through the logger which uses winston and that did log to cloudwatch. I was not expecting that to log to NR unless if I missed a commit.

<img width="872" alt="Screenshot 2024-12-20 at 3 13 34 PM" src="https://github.com/user-attachments/assets/ea86ae42-939d-4162-ab78-7d70f8eb481b" />

Reply:
Oh interesting ! I wasn't expecting it to log to NR either, since I was looking at [the browser logs](https://one.newrelic.com/nr1-core/errors-inbox/entity-inbox/MTIxMzM0fEJST1dTRVJ8QVBQTElDQVRJT058MTU4OTAwNTk4NA?account=121334&duration=86400000&filters=%28domain%20IN%20%28%27BROWSER%27%2C%20%27BROWSER%27%29%20AND%20type%20IN%20%28%27APPLICATION%27%2C%20%27MICRO_FRONTEND%27%29%29&state=68595f69-2214-5389-8c0c-5720d9432df3).  But NR is actually picking up everything I guess via this other APM service– so all those "test error from fetch API" were just logging whenever `fetchAPI` was called all night, so that makes sense.

Reply:
Testing with the divisions page now...

Reply:
It seems like newrelic.log is actually not logging , it's only newrelic.noticeError @EdwinGuzman so I'll update the docs to reflect that